.github/workflows/ci.yaml

```diff
@@ -96,88 +96,94 @@ jobs:
         run: go test -race ./... -timeout 2m
 
   cli:
-    runs-on: ubuntu-latest
-    timeout-minutes: 3
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
 
-    services:
-      postgres:
-        image: postgres
-        env:
-          POSTGRES_PASSWORD: postgres
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 2s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-          - 5432:5432
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 10
```

Comment:
Dang, so just TBC, it looks like this will be by far the slowest job now right? (Last run took 4 min, which is 33% slower than the 3 min the test suite takes.)

River core's CI is still within reasonable bounds time-wise, although it's pretty slow already. We can see how bad it could get by looking at River UI though, where the container jobs are _so_ slow that they really do have a substantial effect on the productivity loop.

Reply:
Yeah, I haven't really looked into whether it can be optimized yet, though I suspect it's not caching properly due to the fact the step hasn't yet succeeded.

For the time being I'm not going to make it a required check, we'll see where we're at once it's running successfully.